Readme.md

```diff
@@ -52,6 +52,8 @@ export default {
   parameters: {
     status: {
       type: 'beta', // 'beta' | 'stable' | 'deprecated' | 'releaseCandidate'
+      // 
```

Comment:
What's the blank comment lines for?

Reply:
Oh sorry, at first I tried to just add it as a comment here, I will remove it